packages/api-db-mongodb/src/utility.ts

```diff
@@ -40,7 +40,13 @@ export function richTextToString(blocksString: string, nodes: any = []) {
   const richTextNode = nodes.reduce((string: string, node: any) => {
     if (!node.text && !node.children) return string
     if (node.text) {
-      return `${string} ${node.text}`
+      if (string.endsWith(' ') || node.text.startsWith(' ')) {
```

Comment:
Isn't this here a usecase for https://www.javascripttutorial.net/javascript-string-trim/

Reply:
@roosnic1 can you elaborate?